_data/simulations/moose_psu_1c_STA/meta.yaml

```diff
@@ -0,0 +1,58 @@
+---
+  benchmark:
+    id: 1c
+    version: 1
+
+  metadata:
+    summary: Metadata for the Benchmark1 problems
+    timestamp: 02/20/2017
+    author: PC. Simon
+    email: pjs5523@psu.edu
+    timestamp: "Mon Feb  6 19:47:34 2017 -EST"
```

Comment:
This is an invalid time stamp, need something like, "Wed Feb 22 15:18:26 EST 2017"